### DISCUSSION TITLE
[M-Z] (12 Febbraio 2019) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#441] [Esami & Esoneri M-Z]
# Traccia

<img width="871" alt="image" src="https://github.com/user-attachments/assets/30451bfb-f828-4d68-b1be-31da823b79cb">

**Comment by rimaout** [comment]
La chiusura simmetrica di una relazione $R$ su $A \times B$  è la più piccola relazione simmetrica su $A \times B$ che contiene $R$.

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190212.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190212.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/924aead6-48f7-434c-9d07-0cd0d553e985)
